Zend/zend_types.h

```diff
@@ -550,6 +550,7 @@ struct _zend_ast_ref {
 #define IS_VOID						14
 #define IS_STATIC					15
 #define IS_MIXED					16
+#define IS_NORETURN					17
```

Comment:
"noreturn" is not really a type. It may be better to use a bit in op_array->fn_flags.